com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDCameraFrameHistoryType.cs

```diff
@@ -41,7 +41,9 @@ public enum HDCameraFrameHistoryType
         RayTracedSubSurface,
         /// <summary>Path tracing buffer.</summary>
         PathTracing,
-        /// <summary>Number of history buffers.</summary>
+        /// <summary>Temporal antialiasing history after DoF.</summary>
+        TemporalAntialiasingPostDoF,
+        /// <summary>Velocity magnitude history used for TAA velocity weighting (after DoF).</summary>
```

Comment:
Mismatched comment now :) Should be 

        /// <summary>Number of history buffers.</summary>